webplugin/js/app/sentry-8.39.0.js

```
@@ -0,0 +1,331 @@
+        'Non-Error exception captured with keys: currentTarget',
+    ],
+    /**
+     * If there any error occurs on the start of the widget then it's easy to debug like who is the user
+     * previously it's not attach
+     */
+    initialScope: (scope) => {
+        Sentry &&
+            Sentry.setTags({
+                url: parent.window.location.href,
+                applicationId: applozic._globals.appId,
+            });
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)